src/components/Dashboard.tsx

```diff
@@ -111,13 +109,9 @@ export default function Dashboard({ isAuthorized, userID, data }: DashboardProps
       if (res.ok) {
         const data = await res.json();
         setUser(data[0])
-        setLoading(false)
-      } else {
-        console.log('Error:', res.statusText);
-        setServerError(true)
+        Router.push("/registrationform");
```

Comment:
I think it may be better practice to use `router.push` from the `useRouter` hook rather than importing the uppercase `Router` from next.

Reply:
oh gotcha! I will go back and update those!